### PR TITLE
Add routes for help sections

### LINKED ISSUE #5529: Add direct links to /help
We already have a great page where many questions of newcomers are answered. But at the moment there is no possibility to link to an specific question.

So if you like to refer to a an single question you have to write something like:
>    Go to help, click on Aspects and open the third question

Links to the specific questions would be an great improvement.

### PR DESCRIPTION
**Pull request by svbergerem**
Fixes #5529. I used the code @AugierLe42e wrote in #5603 and fixed the jasmine tests.

### CODE REVIEW

**Comment by AugierLe42e** [comment]
Cool ! Thank you for dealing with that.

File: spec/javascripts/app/views/help_view_spec.js [hunk 1]
```diff
@@ -87,4 +87,51 @@ describe("app.views.Help", function(){
...
+    beforeEach(function() {
+      this.view.render();
+    });
+
+    it('should rewrite the location', function(){
+      var sections = [
+        'account_and_data_management',
+        'miscellaneous'
+      ];
+      spyOn(app.router, 'navigate');
```
Comment (salbertson) [inline spec/javascripts/app/views/help_view_spec.js]: You can add `spyOn` to `predef` in [.jshint.json](https://github.com/diaspora/diaspora/blob/develop/config/.jshint.json) to tell Hound not to comment on this.

Comment (svbergerem) [inline spec/javascripts/app/views/help_view_spec.js]: Thanks! Already did this 2 hours ago in the [next commit](https://github.com/svbergerem/diaspora/commit/c5468f072fb24e4185a8800d11af3206ee8b1d05). ;-)

**Comment by jhass** [comment]
Thank you!

File: spec/javascripts/app/views/help_view_spec.js (hunk 1 above)
Comment (salbertson) [inline spec/javascripts/app/views/help_view_spec.js]: Nice!

**Pull request merged**